src/Utilities/UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -57,7 +57,7 @@ public void GetApiContractReferencesHandlesNonExistingLocation()
         public void GetApiContractReferencesFindsWinMDs()
         {
             string tempDirectory = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
-            string referenceDirectory = Path.Combine(tempDirectory, @"References\Foo\Bar");
+            string referenceDirectory = Path.Combine(tempDirectory, NativeMethodsShared.IsWindows ? @"References\Foo\Bar" : "References/Foo/Bar");
```

Comment:
`Path.Combine(tempDirectory, "References", "Foo", "Bar")` should work everywhere, right?